stock_dropshipping_dual_invoice/tests/test_need_two_invoices.py

```diff
@@ -73,14 +73,30 @@ def test_so_on_delivery_need_one_invoice(self):
         }).create({})
         self.assertIs(False, wizard._need_two_invoices())
 
+    def test_so_mto_need_one_invoice(self):
+        self.so_mto.order_policy = 'picking'
+        self.so_mto.action_button_confirm()
+        procurements = self.so_mto.procurement_group_id.procurement_ids
+        procurements.run()
+        po = procurements.filtered('purchase_id')[:1].purchase_id
+        self.assertTrue(po)
+        po.invoice_method = 'picking'
+        po.signal_workflow('purchase_confirm')
+        picking = po.picking_ids[:1]
```

Comment:
This is personal opinion, but I prefer how I did in the other test. The assertion that the picking is one is crucial: if something goes wrong with the validation of the order, there will be no picking, and I'd like to catch that explicitly. (IIRC I made this mistake while writing the test (i.e. my order was wrong, and there was no picking).

`[:1]` means we accept zero, one, or more pickings, and want to get the first one, or none if there is none. This sounds like a complex behavior with many edge cases, while the only case we want is exactly one picking (not zero, not many).